spyrmsd/__main__.py

```diff
@@ -25,6 +26,14 @@
     parser.add_argument(
         "-n", "--nosymm", action="store_false", help="No graph isomorphism"
     )
+    parser.add_argument(
+        "-g",
+        "--graph-backend",
+        type=str,
+        default=None,
+        choices=spyrmsd.available_backends,
```

Comment:
This is perfectly fine. However in PR I would have been a little more flexible and use the _validate_backend function in graph.py, to have the exact same behavior (and support for aliases like nx, gt, ...). This is a bit more work and verbose, but maybe a nice to have?

Reply:
That's a very good point, thanks! I think if I simply remove "choices" it should work as you suggest, since `set_backend` internally calls `_validate_backend`.

Reply:
I also added a warning filter, in case the backend that is passed is the same as the one that is already set.

Reply:
Yep! If an unsupported backend is given as an input it will be should by the _validate_backend anyway.